pennylane/transforms/decompose.py

```diff
@@ -431,18 +647,29 @@ def circuit():
     ──────╰●────────────────────────────┤
     """
 
-    if gate_set is None:
-        gate_set = set(qml.ops.__all__)
+    _graph_decomps_kwargs_checks(fixed_decomps, alt_decomps)
 
     if isinstance(gate_set, (str, type)):
         gate_set = set([gate_set])
```

Comment:
Might be irrelavant just wondering if this is equivlanet
```suggestion
        gate_set = {gate_set}
```

Reply:
Nice suggestion!